### ISSUE TITLE
Strange Wifi issues with Linksys WRT32X

### ISSUE THREAD

**Issue by sandreas** [divestedcg/Divested-WRT#1]
Hey Tan,

thanks for making divested - this is a really great project. I use a Linksys WRT32X and with either OpenWRT Stock or Divested I'm experiencing strange Wifi issues:

- Wifi 2.4GHz stops working without dropping the connection (only reboot solves this)
- Wifi 5GHz has bandwidth drops every now and then

These are so random, I didn't even notice it was an issue that happens to other people :-) I did a bit of research and it seems to be a "known" problem but not everyone is experiencing this. [I was recommended to use another device](https://news.ycombinator.com/item?id=37871486), because it has something to do with the hardware/firmware.

My Questions:
- Do you know anything about this?
- Why aren't you using github actions for building the images? :-)

BTW: 
Since the Belkin RT3200 / Linksys E8450 is not available in Germany ATM, I now ordered a [Banana Pi R3 - BPI-R3](https://wiki.junicast.de/en/junicast/review/bananapi-BPI-R3). Hopefully I can get rid of the issues with it.

Interesting devices I considered are:
- [GL.inet Flint 2 / MT 6000](https://www.gl-inet.com/products/gl-mt6000/) - to be released end of november
- Belkin RT3200 / Linksys E8450 - most stable wifi of all WRT devices, but not available in Germany
- [Banana Pi R3 - BPI-R3](https://wiki.junicast.de/en/junicast/review/bananapi-BPI-R3) - good but you have to assemble the kit version yourself
- Netgear WAX206 - not available in germany
- Xiaomi AX3600 / X6S - OpenWRT flashing procedure is too fiddly and unclear because of different hardware revisions
- Netgear R7800 - rock solid but no Wifi 6 / ax

**Comment by SkewedZeppelin** [comment]
hm, seems I missed this

you closed it, was it still an issue?

I don't have a wrt32x or wrt3200acm, you'll likely get a better response asking in the forum thread.

**Comment by sandreas** [comment]
> hm, seems I missed this

Nevermind, it's no longer important to me.

> you closed it, was it still an issue?

Yes - it has been an issue for a very long time now. It seems to be a problem with the firmware, but there is no clear reason. There are a lot of reports regarding instabilities all over the internet for this device - some of them are older and reported as fixed, but they arent. Maybe that's why nobody really takes care.

I thought it was a problem with my internet provider until [I read about it on hackernews](https://news.ycombinator.com/item?id=37873717). After investigating I noticed that I was dealing with Wifi issues... It isn't that bad - a normal person probably wouldn't have noticed, but I noticed strange fluctuations in my wifi performance as well as connection drops. 

Since there was not a fix or someone taking care, I did some research and found better and newer devices.

> I don't have a wrt32x or wrt3200acm, you'll likely get a better response asking in the forum thread.

I sold my devices in favor of Banana Pi BPI-R3 and Gl.inet Beryl for traveling. There is also reportings that the Linksys E8450 / Belkin RT3200 is the most stable device for now, so I'd recommend getting one of these instead and get rid of the Linksys 32... devices.
